src/sentry/discover/dashboard_widget_split.py

```diff
@@ -156,9 +157,22 @@ def _get_and_save_split_decision_for_dashboard_widget(
                 offset=None,
                 limit=1,
                 referrer="tasks.performance.split_discover_dataset",
+                transform_alias_to_input_format=True,
             )
 
-            if metrics_query_result.get("data") and len(metrics_query_result["data"]) > 0:
+            has_metrics_data = (
+                metrics_query_result.get("data")
+                # No results were returned at all
+                and len(metrics_query_result["data"]) > 0
+                # All of the aggregates in any possible row return 0
+                and any(
+                    metrics_query_result["data"][row][column] > 0
+                    for column in selected_columns
+                    if is_aggregate(column)
+                    for row in range(len(metrics_query_result["data"]))
+                )
```

Comment:
```suggestion
                and any(
                    metrics_query_result["data"][0][column] > 0
                    for column in selected_columns
                    if is_aggregate(column)
                )
```

nit: we can simplify this to just check `metrics_query_result["data"][0][column]` because we always limit the query to 1 row when checking for splits